spec/spec.md

```diff
@@ -957,7 +957,7 @@ On a disconnection:
 - If the connection is part of the fully open reflections data set, the entry will be removed and the other connection closed as well
 - Otherwise, the entry for the connection will be removed from the half-open reflections data set
 
-Entries in the half open data set should be removed, and the connection closed, if still present in this set  X seconds after being added. Entries in the fully open data set should be removed, and both connections closed, if still present in this list X seconds after being added.
+Entries in the half open data set should be removed, and the connection closed, if still present in this set no less than 30 seconds after being added. Entries in the fully open data set should be removed, and both connections closed, if still present in this list no less than 90 seconds after being added.
```

Comment:
Rationale for 30s same as above.

Rationale for 90s - no specific rationale, but this puts a 2 min cap (30s + 90s) on reflector connections, protecting the server from non-malicious resource exhaustion.